CHANGELOG.md

```diff
@@ -17,6 +17,7 @@
 * Fix #2537: Checking for Readiness of DeploymentConfig
 * Fix #2300: Remove job extensions/v1beta1 from backward compatibiliy interceptor
 * Fix #2514: SharedIndexInformer watches only pods of its own namespace when run in the cluster
+* Fix #2596: Add buildable references for Container, Port and Volume
```

Comment:
This and the previous change  (`Eliminated the use of Doneables and simplified the internal DSL implementation`) belong to the 5.0- SNAPSHOT section.